test/units/testsuite-83.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
Maybe better to make the test a subtest of TEST-19.

Reply:
I'd like to keep this separate for now.

Reply:
Adding new separate tests here has a lot of overhead, due to the setup, so it shouldn't be done unless there's a good reason. This would fit very well as part of TEST-07-PID1 which is also testing similar features.

Reply:
Ok, so let's add it to TEST-07-PID1